Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Customers.h

```diff
@@ -93,27 +97,44 @@ typedef void (^BUYDataOrdersBlock)(NSArray <BUYOrder*> * _Nullable orders, NSErr
 - (BUYRequestOperation *)createCustomerWithCredentials:(BUYAccountCredentials *)credentials callback:(BUYDataCustomerTokenBlock)block;
 
 /**
- *  POST /api/customers/customer_token
- *  Logs in an existing customer
- *  Expects email and password
+ *  PUT /api/customers/:customer_id/activate
```

Comment:
One day, git will be integrated with the language interpreter. It will realize that changes that only move code around—without changing its meaning (like moving method/function declarations/implementations)—won't create these diffs.